tests/unit/test_tutoring_model.py

```diff
@@ -129,6 +129,241 @@ def test_prepare_les_for_ga(learning_style):
     assert result[-1] == "LZ"
 
 
+@pytest.mark.parametrize(
+    "learning_style",
+    [
+        (
+            {
+                "id": 1,
+                "characteristic_id": 2,
+                "perception_dimension": "int",
+                "perception_value": 5,
+                "input_dimension": "vrb",
+                "input_value": 0,
+                "processing_dimension": "ref",
+                "processing_value": 9,
+                "understanding_dimension": "seq",
+                "understanding_value": 11,
+            }
+        ),
+        (
+            {
+                "id": 3,
+                "characteristic_id": 4,
+                "perception_dimension": "sns",
+                "perception_value": 1,
+                "input_dimension": "vis",
+                "input_value": 11,
+                "processing_dimension": "act",
+                "processing_value": 1,
+                "understanding_dimension": "glo",
+                "understanding_value": 3,
+            }
+        ),
+    ],
+)
+def test_prepare_les_for_tyche(learning_style):
+    # Test Tyche with success:
+    list_of_les = []
+    list_of_keys = [
```

Comment:
maybe use from utils.constants

Reply:
also consider parametrizing the test using these abbrevations. This may lead to less code.

Reply:
used parameters defined in utils.constants